server/sonar-server/src/main/java/org/sonar/server/user/ws/UserPropertiesWs.java

```diff
@@ -20,30 +20,27 @@
 package org.sonar.server.user.ws;
 
 import org.sonar.api.server.ws.WebService;
-
-import static org.sonar.api.server.ws.RailsHandler.INSTANCE;
-import static org.sonar.api.server.ws.RailsHandler.addFormatParam;
+import org.sonar.server.ws.RemovedWebServiceHandler;
 
 public class UserPropertiesWs implements WebService {
 
   @Override
   public void define(Context context) {
     NewController controller = context.createController("api/user_properties");
-    controller.setDescription("Manage user properties.");
+    controller.setDescription("Manage user properties. <br/>" +
```

Comment:
we should say the « replacements »: my notifications and I think there are others